schema.sql

```diff
@@ -0,0 +1,12 @@
+/* Database schema to keep the structure of entire database. */
+CREATE TABLE animals (
+CREATE TABLE animals(
+    name varchar(100)
+    id INT,
+    name VARCHAR,
+    date_of_birth DATE,
+    escape_attempts INT,
+    neutered BOOLEAN,
+    weight_kg DECIMAL
+);
+);
```

Comment:
- [ ] Kindly fix this small issue with the animals table, you seem to have nested two create commands by mistake.
- [ ] [ _Optional_ ] I would advise you to create the `id` column an Identity Primary key column instead. That way, it will be the unique identifier for the data in this table and the value would be auto incremented which means that you won't have to specify a unique id everytime you add new data.
``` SQL
    id INT PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
```